# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsGlobalWindowInner.cpp
# Commit: cc3c39aa8fb8
# Full Hash: cc3c39aa8fb8124cca265e84f71a29cbd218b704
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r 2fc9a3d0ad8b -r cc3c39aa8fb8 dom/base/nsGlobalWindowInner.cpp
--- a/dom/base/nsGlobalWindowInner.cpp	Wed Sep 11 16:26:54 2024 +0000
+++ b/dom/base/nsGlobalWindowInner.cpp	Wed Sep 11 16:26:55 2024 +0000
@@ -5932,6 +5932,10 @@
   // Fires an offline status event if the offline status has changed
   FireOfflineStatusEventIfChanged();
 
+  if (mCookieStore) {
+    mCookieStore->FireDelayedDOMEvents();
+  }
+
   if (!aIncludeSubWindows) {
     return NS_OK;
   }